modules/xfem/src/auxkernels/MeshCutLevelSetAux.C

```diff
@@ -0,0 +1,47 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MeshCutLevelSetAux.h"
+#include "InterfaceMeshCutUserObjectBase.h"
+
+registerMooseObject("XFEMApp", MeshCutLevelSetAux);
+
+InputParameters
+MeshCutLevelSetAux::validParams()
+{
+  InputParameters params = AuxKernel::validParams();
+  params.addClassDescription(
+      "Auxiliary Kernel that calcuates level set value using line segments' description.");
```

Comment:
This isn't just for line segments, right?  Would it be better to say:
Calculates signed distance from interface defined by InterfaceMeshCutUserObject